xds/src/main/java/io/grpc/xds/XdsLbPolicies.java

```diff
@@ -24,7 +24,6 @@
   static final String PRIORITY_POLICY_NAME = "priority_experimental";
   static final String CLUSTER_IMPL_POLICY_NAME = "cluster_impl_experimental";
   static final String WEIGHTED_TARGET_POLICY_NAME = "weighted_target_experimental";
-  static final String LRS_POLICY_NAME = "lrs_experimental";
 
```

Comment:
EDS can also be removed.

Reply:
Removed EDS. Btw, our existing way of declaring all LB policy names in XdsLbPolicies is anti-pattern in Java, we may consider getting rid of it. I think using string literals in LB providers would be fine.